NOTICE.txt

```diff
@@ -565,7 +565,7 @@ Contents of "LICENSE":
 
 --------------------------------------------------------------------
 Dependency: github.com/gofrs/uuid
-Version: v3.2.0
+Version: v3.3.0
```

Comment:
any concerns here?

Reply:
no, we only use the `uuid` for log entries, so any changes wouldn't be concerning; and the [diff](https://github.com/gofrs/uuid/compare/v3.2.0...v3.3.0) indicates they only implemented the `fmt.Formatter`.

Reply:
thanks for checking